### PR TITLE
Add a check for character limit for the input text

### PR DESCRIPTION
**Pull request by vimleshtna**
Manually upgraded the version as well as it failed to create version bump PR last time

### CODE REVIEW

File: src/main/scala/uk/gov/nationalarchives/tdr/validation/MetadataCriteria.scala [hunk 1]
```diff
@@ -43,4 +44,5 @@ object ErrorCode {
   val MULTI_VALUE_ERROR = "MULTI_VALUE_ERROR"
   val UNDEFINED_VALUE_ERROR = "UNDEFINED_VALUE_ERROR"
   val REQUIRED_PROPERTY_IS_EMPTY = "REQUIRED_PROPERTY_IS_EMPTY"
+  val TOO_LONG_INPUT_ERROR = "TOO_LONG_INPUT_ERROR"
```
Comment (TomJKing) [inline src/main/scala/uk/gov/nationalarchives/tdr/validation/MetadataCriteria.scala:47]: Could this error be renamed to something like this: `MAX_CHARACTER_LIMIT_INPUT_ERROR`?

File: version.sbt [hunk 2]
```diff
@@ -1 +1 @@
-ThisBuild / version := "0.0.1-SNAPSHOT"
+ThisBuild / version := "0.0.2-SNAPSHOT"
```
Comment (TomJKing) [inline version.sbt:1]: This shouldn't need manually bumping. Will need to look at the build action to see what the problem is with it.

**Review by TomJKing — COMMENTED** [review]
Looks good. Just a couple of minor comments. Thanks

File: src/main/scala/uk/gov/nationalarchives/tdr/validation/MetadataCriteria.scala (hunk 1 above)
Comment (vimleshtna) [reply src/main/scala/uk/gov/nationalarchives/tdr/validation/MetadataCriteria.scala]: Yeah, that looks good to me as well

File: version.sbt (hunk 2 above)
Comment (vimleshtna) [reply version.sbt:1]: Looks like the issue is with the `Version bump` label. I have added it manually. Hopefully, in the next deployment, it will work.

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks

**Pull request merged**